templates/taxi/driver_list.html

```diff
@@ -7,7 +7,11 @@ <h1>Driver list</h1>
       {% for driver in driver_list%}
         <li>
           <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
```

Comment:
Ensure that the `driver-detail` URL pattern expects a `pk` parameter corresponding to the driver's ID. This is necessary for the link to function correctly.